package.json

```diff
@@ -6,6 +6,7 @@
   },
   "scripts": {
     "build:boxel-host": "cd packages/host && pnpm build:production",
+    "clear-caches": "rm -rf node_modules packages/**/node_modules $TMPDIR/embroider",
```

Comment:
this command might work better for finding all the node_modules and deleting them:
```
find . -name 'node_modules' -type d -prune -exec rm -rf {} \;
```

Reply:
thanks, I’ve updated it with that now 😀